content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+[bootstraptoken] creating the "cluster-info" ConfigMap in the "kube-public" namespace
+[addons] Applied essential addon: CoreDNS
+[addons] Applied essential addon: kube-proxy
+
+Your Kubernetes control-plane has initialized successfully!
+
+To start using your cluster, you need to run the following as a regular user:
+
+  mkdir -p $HOME/.kube
+  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+You should now deploy a Pod network to the cluster.
+Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
+  /docs/concepts/cluster-administration/addons/
+
+You can now join any number of machines by running the following on each node
+as root:
+
+  kubeadm join <control-plane-host>:<control-plane-port> --token <token> --discovery-token-ca-cert-hash sha256:<hash>
+```
+
+Untuk membuat kubectl bekerja pada pengguna _non-root_, jalankan perintah-perintah berikut, yang juga merupakan
+bagian dari keluaran `kubeadm init`:
+
+```bash
+mkdir -p $HOME/.kube
+sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+sudo chown $(id -u):$(id -g) $HOME/.kube/config
+```
+
+Alternatif lainnya, jika kamu adalah pengguna `root`, kamu dapat menjalankan:
+
+```bash
+export KUBECONFIG=/etc/kubernetes/admin.conf
+```
+
+Buatlah catatan dari perintah `kubeadm join` yang dihasilkan `kubeadm init`. Kamu
+membutuhkan perintah ini untuk [menggabungkan simpul-simpul ke klaster](#menggabungkan-simpul).
+
+_Token_ digunakan untuk otentikasi mutual antara simpul _control-plane_ dan simpul-simpul yang
+akan bergabung. _Token_ yang didapat di sini bersifat rahasia. Simpan dengan aman, karena siapapun yang memiliki token tersebut
+dapat menambahkan simpul-simpul yang otentik ke klaster. Kamu dapat menampilkan daftar _token_,
+membuat, dan menghapus _token_ dengan perintah `kubeadm token`. Lihat
+[panduan referensi kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm-token/).
+
+### Memasang _add-on_ jaringan Pod {#jaringan-pod}
+
+{{< caution >}}
+Bagian ini berisi informasi penting mengenai penyetelan jejaring dan
+urutan penyebaran.
+Baca seluruh saran ini dengan saksama sebelum melanjutkan.
+
+**Kamu harus menyebarkan
+_add-on_ jaringan Pod berbasis {{< glossary_tooltip text="_Container Network Interface_" term_id="cni" >}}
+(CNI) sehingga Pod dapat berkomunikasi satu sama lain.  
+DNS klaster (CoreDNS) tidak akan menyala sebelum jaringan dipasangkan.**
+
+- Perlu diperhatikan bahwa jaringan Pod tidak boleh tumpang tindih dengan jaringan _host_
+  manapun: kamu akan menemui beberapa masalah jika terjadi tumpang tindih.  
+  (Jika kamu menemukan adanya bentrokan antara jaringan Pod
+  pilihan _plugin_ jaringan dengan jaringan _host_, kamu harus memikirkan blok
+  CIDR yang cocok untuk digunakan, kemudian menggunakannya pada saat `kubeadm init` dengan
+  `--pod-network-cidr` dan sebagai pengganti pada YAML _plugin_ jaringan).
```

Comment:
```suggestion
  `--pod-network-cidr`, atau sebagai penggantinya pada berkas YAML _plugin_ jaringan kamu).
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc